functions-and-scope.js

```diff
@@ -45,21 +35,7 @@ console.log(amountOfCumLaude)
 //     return
 // }
 
-const numbers = [9, 6, 4, 9, 0, 2, 7];
-
-function getCumLaudeAmountByArray(numbers){
-    let amount = 0;
-    for (let i = 0; i < numbers.length; i++) {
-        if(numbers[i] >= 8){
-            amount++;
-        }
 
-    }
-
-    console.log(numbers);
-
-
-}
 
 
```

Comment:
Hier mag je de functie aanroepen, dan wordt hij ook actief. Zie ook  in uitwerkingen vanaf regel 56.